tests/functional/aws-node-sdk/test/multipleBackend/mpuParts/azurePutPart.js

```diff
@@ -20,30 +20,34 @@ let bucketUtil;
 let s3;
 
 function checkSubPart(key, uploadId, expectedParts, cb) {
-    azureClient.listBlocks(azureContainerName, key, 'all', (err, list) => {
-        assert.equal(err, null, 'Expected success, got error ' +
-        `on call to Azure: ${err}`);
-        const uncommittedBlocks = list.UncommittedBlocks;
-        const committedBlocks = list.CommittedBlocks;
-        assert.strictEqual(committedBlocks, undefined);
-        uncommittedBlocks.forEach((l, index) => {
-            assert.strictEqual(l.Name, getBlockId(uploadId,
-                expectedParts[index].partnbr, expectedParts[index].subpartnbr));
-            assert.strictEqual(l.Size, expectedParts[index].size.toString());
+    azureClient.getContainerClient(azureContainerName)
+        .getBlockBlobClient(key)
+        .getBlockList('all').then(list => {
+            const uncommittedBlocks = list.uncommittedBlocks;
+            const committedBlocks = list.committedBlocks;
+            assert.strictEqual(committedBlocks, undefined);
+            uncommittedBlocks.forEach((l, index) => {
+                assert.strictEqual(l.name, getBlockId(uploadId,
+                    expectedParts[index].partnbr, expectedParts[index].subpartnbr));
+                assert.strictEqual(l.size, expectedParts[index].size.toString());
+            });
+            cb();
+        }, err => {
+            assert.equal(err, null, 'Expected success, got error ' +
+                `on call to Azure: ${err}`);
+            cb();
```

Comment:
Why not:
```suggestion
        }, cb);
```

.. as the error message will stringify and maybe reduce the information carried by the error object?